src/components/Video/Video.tsx

```diff
@@ -1,60 +1,115 @@
-import useSwr from 'swr';
+import React, {
```

Comment:
As we need the video data on the focused video player view, I was able to move VideoContent back to being known as Video.